components/CredentialDashboard.vue

```diff
@@ -26,9 +26,14 @@
  * Copyright (c) 2018-2022 Digital Bazaar, Inc. All rights reserved.
  */
 import {computed, ref, toRef, watch} from 'vue';
+import {formatString, getValueFromPointer} from '../lib/helpers.js';
 import {createEmitExtendable} from '@digitalbazaar/vue-extendable-event';
 import CredentialsList from './CredentialsList.vue';
 import SearchBox from './SearchBox.vue';
+import vcViewConfigurations from '../lib/vcViewConfigurations.json';
```

Comment:
This should instead be incorporated into the main frontend config somewhere in this module's config namespace.

Reply:
Updated with commit 400b31e and 7b34a3d